src/model/graph.ts

```diff
@@ -101,5 +103,66 @@ export class Graph implements GraphInterface {
     }
   }
 
+  toProtobuf(): protobuf.Graph {
+    let edges: { [key: string]: protobuf.Edge } = {};
```

Comment:
Try to use `const` instead of `let` and `var` everywhere as it makes the code a bit "safer"